docs/_static/css/custom.css

```diff
@@ -52,3 +52,18 @@ div.sponsor-div-box {
    font-size: 0.9rem;
    text-align: center;
  }
+
+ /* Fix version switcher button margin bottom */
+ button.version-switcher__button {
+   margin-bottom: 0em;
+ }
+
+ /* Fix scrollbars on primary sidebar */
+ .bd-sidebar-primary .sidebar-end-items {
+   margin-top: unset;
+ }
+
+ /* Fix Sphinx-Glr page bottom border */
+ p.sphx-glr-signature a.reference.external {
+   padding-bottom: 0px;
+ }
```

Comment:
Fixes the inconsistent bottom border on the Sphinx Gallery Examples page.

**Before**
![image](https://user-images.githubusercontent.com/50991099/203375721-05bceb85-8bbf-4caa-8688-e41d860f919b.png)

**After**
![image](https://user-images.githubusercontent.com/50991099/203376117-66589cff-ffc5-4831-97d8-845b9cb33223.png)